flag/standard/harb_kotf/map.xml

```diff
@@ -121,15 +121,7 @@
     </portal>
 </portals>
 <flags>
-    <flag id="flag"
-          name="Flag"
-          color="white"
-          shared="true"
-          points-rate="1"
-          pickup-kit="flag-kit"
-          drop-kit="reset-health"
-          flag-proximity-metric="none"
-          show-respawn-on-pickup="true">
+    <flag id="flag" name="Flag" color="white" shared="true" points-rate="1" pickup-kit="flag-kit" drop-kit="reset-health" flag-proximity-metric="none" show-respawn-on-pickup="true">
```

Comment:
I've usually left each attribute line by line in KotF just for legibility reasons, but it's not a big deal

Reply:
I mean you could argue that most of these changes are just between different ways of doing the same thing.